airflow/www/views.py

```diff
@@ -486,16 +486,22 @@ def rendered(self):
         dag_id = request.args.get('dag_id')
         task_id = request.args.get('task_id')
         execution_date = request.args.get('execution_date')
+        root = request.args.get('root', '')
+
         dttm = pendulum.parse(execution_date)
         form = DateTimeForm(data={'execution_date': dttm})
-        root = request.args.get('root', '')
-        dag = dagbag.get_dag(dag_id)
+
+        dag_model = DagModel.get_dagmodel(dag_id)
+        dag = dag_model.get_dag()
```

Comment:
The other PRs have a `flash()` if dag_model isn't found - should we have that here too?

Reply:
Thanks, I added the flash message.